test/testsuites/operatorhubtests.go

```diff
@@ -90,18 +90,12 @@ func testDisableAllEnableOne(t *testing.T) {
 	err = toggle(t, 1, false, true)
 	require.NoError(t, err, "Error updating cluster OperatorHub")
 
+	err = checkCreated(1, namespace)
+	assert.NoError(t, err, "Default CatalogSource was not created")
+
 	err = checkCatsrcIsPresent(helpers.DefaultSources[0].Name, namespace)
```

Comment:
`checkCreated` instead of `checkCatsrcIsPresent` might be a better idea so that this is polled for too.